src/Game/zLasso.cpp

```diff
@@ -21,22 +180,159 @@
 #pragma GLOBAL_ASM("asm/Game/zLasso.s", "zLasso_Update__FP6zLassoP4xEntf")
 
 // func_8009D794
+#if 1
 #pragma GLOBAL_ASM("asm/Game/zLasso.s", "zLasso_InitTimer__FP6zLassof")
+#else
+void zLasso_InitTimer(zLasso *lasso, float f)
+{
+    lasso->secsTotal = f;
+    lasso->secsLeft = f;
+    lasso->crRadius = zLasso_f_0_0;
+    lasso->tgRadius = zLasso_f_0_0;
+    lasso->stRadius = zLasso_f_0_0;
+    lasso->crSlack = zLasso_f_0_0;
+    lasso->tgSlack = zLasso_f_0_0;
+    lasso->stSlack = zLasso_f_0_0;
+    // iModelTagEval(RpAtomic *, xModelTag *, RwMatrixTag *, xVec3 *);
+    iModelTagEval(
+        lasso->model->Data, &lasso->tag, lasso->model->Mat, &lasso->crCenter);
+    xVec3Copy(&lasso->anchor, &lasso->model->Scale);
+    xVec3SubFrom(&lasso->crCenter, &lasso->anchor);
+    xVec3Copy(&lasso->honda, &lasso->crCenter);
+    lasso->lastDist = zLasso_f_0_0;
+    lasso->currDist = zLasso_f_0_0;
+    xVec3Init(&lasso->stNormal, zLasso_f_0_0, zLasso_f_1_0, zLasso_f_0_0);
+    xVec3Init(&lasso->tgNormal, zLasso_f_0_0, zLasso_f_1_0, zLasso_f_0_0);
+    xVec3Init(&lasso->crNormal, zLasso_f_0_0, zLasso_f_1_0, zLasso_f_0_0);
+    xVec3Copy(&lasso->lastRefs[0], &lasso->crCenter);
+    xVec3Copy(&lasso->lastRefs[1], &lasso->crCenter);
+    xVec3Copy(&lasso->lastRefs[2], &lasso->crCenter);
+    xVec3Copy(&lasso->lastRefs[3], &lasso->crCenter);
+    xVec3Copy(&lasso->lastRefs[4], &lasso->crCenter);
+    lasso->reindex[0] = 0;
+    lasso->reindex[1] = 1;
+    lasso->reindex[2] = 2;
+    lasso->reindex[3] = 3;
+    lasso->reindex[4] = 4;
+}
+#endif
 
 // func_8009D8D4
+#if 0
```

Comment:
This function should have the `#if 0` statement and `#pragma GLOBAL_ASM` statement removed if it's matching. Additionally the assembly for it should be removed from the .text section of `zLasso.s` so that the progress tracker will count it correctly